styles/widgets/common/gridBase.less

```diff
@@ -194,13 +194,21 @@
             }
         }
 
-        .dx-@{widgetName}-rowsview .dx-row .dx-command-drag {
+        .dx-@{widgetName}-rowsview .dx-data-row .dx-command-drag {
             cursor: move;
             vertical-align: middle;
             padding: 0;
         }
     }
 
+    .dx-sortable-dragging .dx-@{widgetName}-rowsview .dx-data-row .dx-command-drag {
```

Comment:
**Default style** for dx-command-drag icon - (.dx-@{widgetName}-rowsview .dx-data-row .dx-command-drag).

1. This style overrides default behavior (**Default style**) for dragging state.
2. Style in .dx-sortable-dragging > div > .dx-gridbase-container > .dx-datagrid-rowsview does not affect on dx-command-drag icon because of **Default style**

Reply:
Add test for this case.

Reply:
Done